Server/src/services/tracker.service.ts

```diff
@@ -100,61 +46,4 @@ export default class TrackerService {
 			trackerId
 		})
 	}
-
-	/**
-	 * TODO: Define internal schema for data? Where?
-	 * Log new data received by a tracker (wrapper to call from tracker endpoints,
-	 * because they cannot "know" what vehicle they are on)
-	 * @param trackerId id of the `Tracker´
-	 * @param timestamp creation timestamp of the log
-	 * @param position current position
-	 * @param heading heading of the vehicle in degree (0-359)
-	 * @param speed speed of the vehicle in kmph
-	 * @param battery battery voltage of the tracker in V
-	 * @param data data received by a tracker
-	 * @returns a new entry `Log` if successful, `null` otherwise
-	 */
-	public static async appendTrackerLog(
```

Comment:
Not sure, if this logic is needed, because of how the trackers work. That is something the hardware team definitely should look on.